src/app/BCP47Tag.svelte

```diff
@@ -0,0 +1,20 @@
+<script lang="ts">
+  import bcp47Parser from "bcp-47";
+
+  const TAG_INPUT_ID = "tag-input";
+
+  let bcp47Tag = "";
+  $: schema = bcp47Parser.parse(bcp47Tag);
+  $: invalidTag = bcp47Tag !== "" && schema.language === null;
+</script>
+
+<style>
+  .error {
+    color: var(--red);
+  }
+</style>
+
+<input id={TAG_INPUT_ID} bind:value={bcp47Tag} />
```

Comment:
Okay... so... I have a lot of ideas on how this can be improved, BUT I don't think it's necessary for this PR. Again, the reason is that #52 is supposed to be temporary.

For one, a Svelte component that sets its own `id` forbids it from being reusable. So this element, as is, cannot be made reusable :/ Again, this component should be deleted once we have a nice language selector, but something to keep in mind in the future.

Another thing, is that, if you want to target elements in Cypress tests, the best practice is to use [`data-cy` attribute](https://docs.cypress.io/guides/references/best-practices.html#Selecting-Elements).

I have many more comments, but it's not worth it for a component I think will be ideally deleted soon 😅 

Here's a simple suggestion though:

```suggestion
<input id={TAG_INPUT_ID} bind:value={bcp47Tag} data-cy={TAG_INPUT_ID} />

```

Reply:
Cool, I didn't try to create a reusable input component cause I thought we are still looking for a UI library

Reply:
I played around with the Svelte playground, and this is what I came up with in terms of a reuable component. Again, not necessary now, but just for fun: https://svelte.dev/repl/6e100f3fc52944b982ce3f41c07a0bed?version=3.26.0

Reply:
Ahhh, that is pretty neat